src/pcap.rs

```diff
@@ -29,7 +29,7 @@ impl IpLog {
         const VERSION_MINOR: u16 = 4;
         const LINKTYPE_RAW: u32 = 101;
 
-        let try = || -> io::Result<_> {
+        let r#try = || -> io::Result<_> {
```

Comment:
This should probably just be called something else.

Reply:
You right, didn't notice, will change :)